packages/react-composites/src/composites/CallComposite/components/SidePane/usePeoplePane.tsx

```diff
@@ -337,20 +651,43 @@ export const usePeoplePane = (props: {
       </>
     );
   }, [
+    muteAllPromptLabels,
+    showMuteAllPrompt,
+    localeStrings.forbidOthersAudioDialogTitle,
+    localeStrings.forbidOthersAudioDialogContent,
+    localeStrings.forbidOthersAudioConfirmButtonLabel,
+    localeStrings.forbidOthersAudioCancelButtonLabel,
+    localeStrings.permitOthersAudioDialogTitle,
+    localeStrings.permitOthersAudioDialogContent,
```

Comment:
You should be able to just make this dependant on localStrings you can see from this example in the CaptionsMoreButton component in composite 
![image](https://github.com/user-attachments/assets/07db835e-d774-4562-aa59-7c7b28ca24b7)

Reply:
These useMemo dependencies are auto generated. Not sure if we want to change it manually.

Reply:
Go with existing pattern, will revert new changes.